tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -263,13 +263,7 @@ def diff_strings(a: str, b: str, *, use_loguru_colors: bool = False) -> str:
 
 
 def genai_gen_text(model, tokenizer, question, max_new_tokens, skip_question, use_chat_template=False):
-    if use_chat_template:
-        model.start_chat()
-        result = model.generate(question, do_sample=False, max_new_tokens=max_new_tokens)
-        model.finish_chat()
-        return result
-    else:
-        return model.generate(question, do_sample=False, max_new_tokens=max_new_tokens, apply_chat_template=False)
+    return model.generate(question, do_sample=False, max_new_tokens=max_new_tokens, apply_chat_template=use_chat_template)
```

Comment:
I believe it is out of scope of this PR, but what if model does not have chat template? Decision about correct tool configuration is user responsibility?